keylime/cloud_verifier_tornado.py

```diff
@@ -103,6 +103,11 @@ def _from_db_obj(agent_db_obj):
     agent_dict = {}
     for field in fields:
         agent_dict[field] = getattr(agent_db_obj, field, None)
+
+    # add default fields that are ephemeral
+    for key in list(exclude_db.keys()):
```

Comment:
Is the `list()` required (I see that is also later in line 513)?

Otherwise LGTM.